src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/OnlineHearingService.java

```diff
@@ -177,14 +175,15 @@ private Decision getDecision(String onlineHearingId, long caseId) {
                             sscsCaseDeails.getData().getCaseReference(),
                             sscsCaseDeails.getId(),
                             getDecision(onlineHearing.getOnlineHearingId(), sscsCaseDeails.getId()),
-                            new FinalDecision(sscsCaseDeails.getData().getDecisionNotes()), hasFinalDecision);
```

Comment:
We should really have a COR Appellant object with just the fields we need rather than exposing the Appellant object we get back from CCD